### ISSUE TITLE
Tutorial 11 -- loss function of Node2Vec class 

### ISSUE THREAD

**Issue by xianhong** [AntonioLonga/PytorchGeometricTutorial#14]
Reading the loss function of Node2Vec class (https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/nn/models/node2vec.html) , i feel the return value of "pos_loss + neg_loss" does not seem to take into account of 'num_negative_samples'. Shouldn't the loss function return "pos_loss + neg_loss * self.num_ negative_samples" instead?
